<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#ffce9d;</string>
  </property>
  <widget class="QPushButton" name="btnPdf">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:9px; background-color:blue;color:white; font-size:15px; font-weight: bold;</string>
   </property>
   <property name="text">
    <string>gerar pdf</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAtualizar">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>310</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:9px; background-color:blue;color:white; font-size:15px; font-weight: bold;</string>
   </property>
   <property name="text">
    <string>Alterar Contato</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExcluir">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>310</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:9px; background-color:blue;color:white; font-size:15px; font-weight: bold;</string>
   </property>
   <property name="text">
    <string>Excluir contato</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:30px; text-align:center; font-weight: bold;color:white;</string>
   </property>
   <property name="text">
    <string>Cadastros</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:9px; background-color:blue;color:white; font-size:15px; font-weight: bold;</string>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabelaContatos">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>691</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color:white;
border-radius: 9px;</string>
   </property>
   <column>
    <property name="text">
     <string>Nome</string>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nome</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Telefone</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tipo Telefone</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
